Model/Transaction.php

```diff
@@ -106,6 +107,20 @@ public function getChildTransactions()
         return $this->_aChildTransactions;
     }
 
+    /**
+     * Returns true is transaction's payment method is Payment on Invoice or Payment in Advance
+     *
+     * @return bool
+     *
+     * @since 1.3.0
+     */
+    public function isPoiPiaPaymentMethod()
+    {
+        $sPaymentId = $this->getPaymentType();
```

Comment:
you can just check if the $this->getPaymentType() === PoiPiaTransaction::NAME

Reply:
getPaymentType() returns our database key, which is not the same as PoiPiaTransaction::NAME.